Test/Unit/Visitor/Arithmetic.php

```diff
@@ -65,7 +65,7 @@ public function case_visitor_exhaustively()
                     new Regex\Visitor\Isotropic(
                         new LUT\Sampler\Random()
                     ),
-                    9
+                    7
```

Comment:
I tried with `8` which produce a reasonable test suite (~200 seconds AFAIR) but ended up getting the #42 bug :/